src/main/java/dinkplugin/util/Utils.java

```diff
@@ -32,9 +36,29 @@ public class Utils {
 
     public static final String WIKI_IMG_BASE_URL = "https://oldschool.runescape.wiki/images/";
 
+    public final Set<ChatMessageType> CLAN_NOTIFICATIONS = Collections.unmodifiableSet(
+        EnumSet.of(
+            ChatMessageType.FRIENDSCHATNOTIFICATION,
+            ChatMessageType.CLAN_MESSAGE,
+            ChatMessageType.CLAN_GUEST_MESSAGE,
+            ChatMessageType.CLAN_GIM_MESSAGE
+        )
```

Comment:
bit of a shotgun approach (since exact message type is unknown), but should be fine since the clan notification is only considered if the game message has already indicated a pet drop in the tick